src/circuit/gadget/ecc/chip.rs

```diff
@@ -256,6 +253,7 @@ impl EccChip {
         // TODO: Move this into mul::Config.
         let mul_overflow =
             mul::overflow::Config::configure(meta, range_check, advices[6..9].try_into().unwrap());
+        let mul_complete = mul::complete::Config::configure(meta, advices[9], add);
```

Comment:
This was placed after `mul_overflow`, whereas prior to this PR it was configured before. At a minimum, reorder these two commits so `mul_complete` is refactored before `mul_overflow`.